README.md

```diff
@@ -569,12 +569,12 @@ When compiling a directory `--source-map` can either be a boolean value or a dir
 
 node-sass supports different configuration parameters to change settings related to the sass binary such as binary name, binary path or alternative download path. Following parameters are supported by node-sass:
 
-Variable name    | .npmrc parameter | Process argument   | Value
------------------|------------------|--------------------|------
-SASS_BINARY_NAME | sass_binary_name | --sass-binary-name | path
-SASS_BINARY_SITE | sass_binary_site | --sass-binary-site | URL
-SASS_BINARY_PATH | sass_binary_path | --sass-binary-path | path
-SASS_BINARY_DIR  | sass_binary_dir  | --sass-binary-dir  | path
+Variable name    | .npmrc parameter | Process argument   | Value | Example
+-----------------|------------------|--------------------|-------|----------------------------
+SASS_BINARY_NAME | sass_binary_name | --sass-binary-name | path  | win32-x64-57
```

Comment:
binary name includes the `.node` extension

https://github.com/sass/node-sass/blob/3331891a89fc65b91b2c184f9159ec189a80fd66/test/runtime.js#L29